lib/src/DOtherSide.cpp

```diff
@@ -287,12 +288,12 @@ void dos_qguiapplication_download_image(const char *imageSource, const char *fil
 
 void dos_qguiapplication_download_imageByUrl(const char *url, const char *filePath)
 {
-    auto manager = new QNetworkAccessManager;
-    manager->setAutoDeleteReplies(true);
-    QObject::connect(manager, &QNetworkAccessManager::finished, [manager, filePath](QNetworkReply *reply) {
+    static thread_local QNetworkAccessManager manager;
+    manager.setAutoDeleteReplies(true);
+    QObject::connect(&manager, &QNetworkAccessManager::finished, [path = QString(filePath)](QNetworkReply *reply) mutable {
```

Comment:
Getting a warning here: `Initialized lambda captures are a C++14 extension`

Reply:
Ok, will redo for c++11.